source/connect/csot.txt

```diff
@@ -15,32 +15,37 @@ Limit Server Execution Time
    :values: reference
  
 .. meta::
-   :keywords: error, blocking, thread, task
+   :keywords: error, blocking, thread, task, code examples
 
 Overview
 --------
 
-When you use the {+driver-short+} to perform a server operation, you can also
+In this guide, you can learn how to use the ``timeout_ms`` option to set a 
+timeout for server operations. 
+
+When you use the {+driver-short+} to perform a server operation, you can
 limit the duration allowed for the server to finish the operation. To do so,
 specify a **client-side operation timeout (CSOT)**. The timeout applies to all
 steps needed to complete the operation, including server selection, connection
-checkout, and server-side execution. When the timeout expires, the
-{+driver-short+} raises a timeout exception.
+checkout, and server-side execution. If the timeout expires before the operation 
+completes, the {+driver-short+} raises a timeout exception.
 
-timeout_ms Option
------------------
+``timeout_ms`` Option
+---------------------
 
 To specify a timeout when connecting to a MongoDB deployment, set the
 ``timeout_ms`` connection option to the timeout length in milliseconds. You can
-do this in two ways: by passing an argument to the ``Mongo::Client`` constructor
-or through a parameter in your connection string.
+specify a timeout value in two ways: 
 
-The following code examples use the ``timeout_ms`` client option or the ``timeoutMS``
-connection string option to specify a timeout of 30 seconds:
+- Pass the ``timeout_ms`` client option to the ``Mongo::Client`` constructor.
+- Pass the ``timeoutMS`` connection string option as a parameter to your connection string.
+
+Select from the following tabs to view examples for how to use the ``timeout_ms`` client
+option or the ``timeoutMS`` connection string option to specify a timeout of 30 seconds:
 
 .. tabs::
 
-   .. tab:: MongoClient
+   .. tab:: Mongo::Client
```

Comment:
```suggestion
   .. tab:: Client Option
```

Reply:
I'll leave as is to stay consistent with the tab naming. Naming the other tab 'Connection String Option' would be too long